tests/adapters/shell/terminal_client_test.py

```diff
@@ -198,6 +226,32 @@ class TelnetClientTest(TerminalClientTest):
     client = TelnetClient
     port = 10011
 
+    @patch('netman.adapters.shell.telnet._connect')
+    @patch('netman.adapters.shell.telnet.TelnetClient._login', Mock())
+    def test_changing_default_connect_timeout(self, connect_method_mock):
+        TelnetClient(**self._get_some_credentials())
+        self.assertEqual(60, connect_method_mock.call_args[0][2])
```

Comment:
The _connect method has 2 other paramteter that i don`t want to care about, therefore i just checked the one that concerns me

    connect_method_mock.assert_caled_once_with(user,password,60)